CONTRIBUTING.md

```diff
@@ -185,16 +185,30 @@ Here is the svg for the Adobe Photoshop icon as an example:
 
 Icon metadata should be added to the `_data/simple-icons.json` file. Each icon in the array has three required values:
 
-  * The `title` of the new SVG.
-  * A `hex` color value that matches the brand's primary color. All uppercase and without the `#` pound symbol.)
-  * The `source` URL of the logo being used. There are [more details below](#source-guidelines).
+* The `title` of the new SVG.
+* A `hex` color value that matches the brand's primary color. All uppercase and without the `#` symbol.
+* The `source` URL of the logo being used. There are [more details below](#source-guidelines).
 
-Additionally, there are also optional fields that may provided for an icon:
+Here is the object of a fictional brand as an example:
 
-- The `guidelines` may be used to specify the URL of the brand's guidelines/presskit/etc. This is useful if the SVG file was sourced from a different place.
-- The `license` may be used to record the license under which the icon is available. This is an object with a `type` and `url`. The `type` should be an [SPDX License ID](https://spdx.org/licenses/) or `"custom"`, the `url` is optional unless the `type` is `"custom"`.
+```json
+{
+    "title": "A Fictional Brand",
+    "hex": "123456",
+    "source": "https://www.a-fictional-brand.org/logo"
+}
+```
 
-Here is the object of a fictional brand as an example:
+Make sure the icon is added in alphabetical order. If you're in doubt, you can always run `npm run our-lint` - this will tell you if any of the JSON data is in the wrong order.
+
+#### Optional Data
```

Comment:
My original intention was for this to be it's own section entirely, at the same level as "Requesting an Icon".

Reply:
Hmm, not sure how well that works with the current structural approach we have :thinking: This information seems to me to be at the same level as e.g. the [SVG Filename Convention](https://github.com/simple-icons/simple-icons/blob/355454cb6caa02aba70638631c557d4e06205710/CONTRIBUTING.md#svg-filename-convention) and [Source Guidelines](https://github.com/simple-icons/simple-icons/blob/355454cb6caa02aba70638631c557d4e06205710/CONTRIBUTING.md#source-guidelines) below. And I would definitely find it strange if this is a higher-level section than [6. Update the JSON Data for SimpleIcons.org](https://github.com/simple-icons/simple-icons/blob/355454cb6caa02aba70638631c557d4e06205710/CONTRIBUTING.md#6-update-the-json-data-for-simpleiconsorg).

If your concern is that it is easy to miss, as per [another comment of mine in this thread](https://github.com/simple-icons/simple-icons/pull/5068#discussion_r580233976), we could update [the Table of Contents](https://github.com/simple-icons/simple-icons/blob/355454cb6caa02aba70638631c557d4e06205710/CONTRIBUTING.md#table-of-contents) and include these lower-level subsections.

Reply:
My reasoning for suggesting we have a separate section for it, at least for now, is that we don't just need contributors adding the additional entries for new icons, we also need help adding it to all our existing icons and a separate section would give it more prominence and not just confine it to the workflow of adding a new icon.

Including the lower level sections in the ToC may be a good compromise, though.

Reply:
> My reasoning for suggesting we have a separate section for it, at least for now, is that we don't just need contributors adding the additional entries for new icons, we also need help adding it to all our existing icons and a separate section would give it more prominence and not just confine it to the workflow of adding a new icon.

Ah, I see now, makes sense. Still not sure if a dedicated level-2 header makes sense though mainly as it would result in this section being somewhere completely different from where it _should_ be (unless, perhaps, we restructure the entire document).

One other thing we can consider that is at least a little closer to your original intention is to somehow include a reference to this section between [Adding or Updating an Icon](https://github.com/simple-icons/simple-icons/blob/86c8dd25a7f7cd151c585fb640ce98493bb65cc1/CONTRIBUTING.md#adding-or-updating-an-icon) and [1. Identify Official Logos and Colors](https://github.com/simple-icons/simple-icons/blob/86c8dd25a7f7cd151c585fb640ce98493bb65cc1/CONTRIBUTING.md#1-identify-official-logos-and-colors).


> Including the lower level sections in the ToC may be a good compromise, though.

I recommend we leave that to a separate Pull Request though.